src/transactions/TransactionFrame.cpp

```diff
@@ -375,41 +376,48 @@ TransactionFrame::commonValidPreSeqNum(AbstractLedgerTxn& ltx, bool chargeFee,
         (protocolVersionStartsFrom(ledgerVersion, ProtocolVersion::V_13) &&
          mEnvelope.type() == ENVELOPE_TYPE_TX_V0))
     {
-        getResult().result.code(txNOT_SUPPORTED);
+        // getResult().result.code(txNOT_SUPPORTED);
```

Comment:
Please remove the commented out code in this method.

Reply:
Thanks for the catch, removed.